engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/type/Set.scala

```diff
@@ -650,108 +650,4 @@ object Set {
-    override val diagnostics: DiagnosticStorage = DiagnosticStorage()
-  ) extends Set
-      with IRKind.Primitive {
-    override protected var id: Identifier = randomId
-
-    /** Creates a copy of `this`.
-      *
-      * @param left        the left operand
-      * @param right       the right operand
-      * @param location    the source location that the node corresponds to
-      * @param passData    the pass metadata associated with this node
-      * @param diagnostics compiler diagnostics for this node
-      * @param id          the identifier for the new node
-      * @return a copy of `this`, updated with the specified values
-      */
-    def copy(
-      left: Expression                     = left,
-      right: Expression                    = right,
-      location: Option[IdentifiedLocation] = location,
-      passData: MetadataStorage            = passData,
-      diagnostics: DiagnosticStorage       = diagnostics,
-      id: Identifier                       = id
-    ): Subtraction = {
-      val res = Subtraction(left, right, location, passData, diagnostics)
-      res.id = id
-      res
-    }
-
-    /** @inheritdoc */
-    override def duplicate(
-      keepLocations: Boolean   = true,
-      keepMetadata: Boolean    = true,
-      keepDiagnostics: Boolean = true,
-      keepIdentifiers: Boolean = false
-    ): Subtraction =
-      copy(
-        left = left.duplicate(
-          keepLocations,
-          keepMetadata,
-          keepDiagnostics,
-          keepIdentifiers
-        ),
-        right = right.duplicate(
-          keepLocations,
-          keepMetadata,
-          keepDiagnostics,
-          keepIdentifiers
-        ),
-        location = if (keepLocations) location else None,
-        passData = if (keepMetadata) passData.duplicate else MetadataStorage(),
-        diagnostics =
-          if (keepDiagnostics) diagnostics.copy else DiagnosticStorage(),
-        id = if (keepIdentifiers) id else randomId
-      )
-
-    /** @inheritdoc */
-    override def setLocation(
-      location: Option[IdentifiedLocation]
-    ): Subtraction = copy(location = location)
-
-    /** @inheritdoc */
-    override def mapExpressions(
-      fn: Expression => Expression
-    ): Subtraction = {
-      copy(left = fn(left), right = fn(right))
-    }
-
-    /** @inheritdoc */
-    override def toString: String =
-      s"""
-         |`type`.Set.Subtraction(
-         |left = $left,
-         |right = $right,
-         |location = $location,
-         |passData = ${this.showPassData},
-         |diagnostics = $diagnostics,
-         |id = $id
-         |""".toSingleLine
-
-    /** @inheritdoc */
-    override def children: List[IR] = List(left, right)
-
-    /** @inheritdoc */
-    override def showCode(indent: Int): String =
-      s"(${left.showCode(indent)} \\ ${right.showCode(indent)})"
-  }
-
-  object Subtraction extends Info {
-    override val name: String = "\\"
```

Comment:
Removed because [it clashes with new lambda syntax](https://github.com/enso-org/enso/issues/6165#issuecomment-1495778471).